### DISCUSSION TITLE
Identify Js files that load and change the behaviour of the page when you interact with it

### DISCUSSION THREAD

**Discussion by jorgedevillalba** [elementor/elementor#22814] [Developers Community]
Hi, I have one 2 Steps form in one page that shows boths steps and only hides the second one when the user interact with the page.

As my cache plugin Delays the Js files execution, I need to know the Js files that load when the user interacts with the page so I can exclude them for delaying execution.

I can not identify those files so I need help to identify them. In fact, I am not sure if that behaviour depends on any Js file or something related with the user´s javascript browser.

This is the page:
https://www.autocaresdiego.com/0-prueba-ver-carga-formulario

I need any help or solution. Thanks

**Comment by VerdiH** [comment]
Hi @jorgedevillalba I hope someone form the Developers Community will see your message. It's only soft launched recently so there's not yet may people aware. If it takes too long, feel free to ask your question at our Global Community on FaceBook as well https://www.facebook.com/groups/Elementors/

I hope you find your answer soon!
